components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -0,0 +1,139 @@
+[Sample seed lists](https://github.com/habitat-sh/on-prem-builder/tree/master/package_seed_lists) exist for the following scenarios:
+
+* Full `core`: the full contents of the upstream `core` origin. The x86_64 Linux set
+    expands to 12GB, the Linux kernel2 set to 1GB, and the Windows set to 3.5GB.
+* Core deps: a subset of `core` consisting of commonly-used buildtime dependencies.
+* Effortless: packages used to start with the [Effortless pattern](https://github.com/chef/effortless). A complete Effortless implementation requires the contents of both the `stable` and the `unstable` channel.
+
+#### Download packages from public origins
+You must be on an Internet-connected host to download packages from the *public* [Builder](https://bldr.habitat.sh/#/pkgs/core)
+service.
+
+Use the `hab pkg download` command with a seed list `</path/to/seed_list>` to download packages for
+your desired architecture `<arch>` from a channel `<channel>` to a directory `<artifact-dir>`:
+
+```shell
+HAB_AUTH_TOKEN=<your_public_Builder_instance_token> hab pkg download --target <arch> --channel <channel> --file </path/to/seed_list> --download-directory <artifact-dir>
+```
+
+For example, to use the Effortless seed list to download `x86_64-linux` packages from the
+`stable` channel to the `builder_bootstrap` directory:
+
+```shell
+HAB_AUTH_TOKEN=<your_public_Builder_instance_token> hab pkg download --target x86_64-linux --channel stable --file package_seed_lists/effortless_x86_64-linux_stable --download-directory builder_bootstrap
+```
+
+#### Bulk-upload packages to on-premises Habitat Builder
+Make sure the directory `<artifact-dir>` and its contents are on a host that can
+access your on-premises Habitat Builder at `https://<your_automate_fqdn>/bldr`.
+
+1. If your on-premises Habitat Builder is using a self-signed certificate, copy the SSL
+   public key certificate chain into `/hab/cache/ssl` on the host from which you will
+   perform the bulk upload to Habitat Builder.
+1. Run the `bulkupload` command to upload artifacts from `<artifact-dir>` to the
+   `<channel>` channel in on-premises Habitat Builder:
+
+   ```shell
+   HAB_AUTH_TOKEN=<your_on-prem_Builder_instance_token> hab pkg bulkupload --url https://<your_automate_fqdn>/bldr/v1 --channel <channel> <artifact-dir> --auto-create-origins
+   ```
+   The `--auto-create-origins` flag will create each origin listed in the
+   `<artifact-dir>/artifacts` directory. If you omit the `--auto-create-origins` flag,
+   use the Habitat Builder UI to create the necessary origins before running the
+   `bulkupload` command.
+
+### Backup Habitat Builder
+Builder backups are performed as part of the [Chef Automate backup process]({{<relref "backup.md">}}).
+
+### Not currently supported
+* high-availability/DR/multinode Builder
+
```

Comment:
Would be great to have a Troubleshooting section, that shows the admin how to turn on Debug logging for the builder services (similar to the current on-prem docs)

Reply:
Added a section on Operating Habitat Builder that points to the relevant documentation for Chef Automate.